### PR TITLE
patch 1.0.0

### PR DESCRIPTION
**Pull request by IhorRome**

### CODE REVIEW

File: src/test/java/core/basesyntax/service/file/FileWriterTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void writeToFIle_correctWrite_Ok() {
+        fileWriter.writeToFile("src/test/resources/writeTest.csv", "Still hate Junit4");
+        File file = new File("src/test/resources/writeTest.csv");
```
Comment (andriiromaniuk) [inline src/test/java/core/basesyntax/service/file/FileWriterTest.java:24]: file pathes should be constants, i think...everywhere.

**Review by andriiromaniuk — CHANGES_REQUESTED** [review]

File: src/test/java/core/basesyntax/service/FruitRecordParserServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    public static void beforeClass() throws Exception {
+        dataValidator = new DataValidator();
+        parser = new FruitRecordParserServiceImpl(dataValidator);
+    }
+
+    @Test
+    public void getRecord_parseCorrectData_Ok() {
+        expected = List.of(R_BANANA_20_DTO);
+        List<FruitRecordDto> actual = parser.getRecord(List.of(TITLE,
+                R_BANANA_20_STRING));
```
Comment (andriiromaniuk) [inline src/test/java/core/basesyntax/service/FruitRecordParserServiceTest.java:36]: Make whole list constant

**Review by andriiromaniuk — CHANGES_REQUESTED** [review]

**Review by andriiromaniuk — APPROVED** [review]

File: src/test/java/core/basesyntax/service/ReportServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        fruitsDao = new FruitsDaoImpl();
+        fileWriter = new FileWriterImpl();
+        reportService = new ReportServiceImpl(fileWriter, fruitsDao);
+    }
+
+    @Test
+    public void createReport_reportFromStorage_Ok() {
+        fruitsDao.add(new Fruit("banana"), 20);
+        fruitsDao.add(new Fruit("apple"), 50);
+        reportService.createReport("src/test/resources/reportTest.csv");
```
Comment (Max-Pochepets) [inline src/test/java/core/basesyntax/service/ReportServiceTest.java:34]:
```suggestion
        reportService.createReport(OUTPUT_PATH);
```

File: src/test/java/core/basesyntax/service/StoreServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    private static FileReader fileReader;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        dataValidator = new DataValidator();
+        parser = new FruitRecordParserServiceImpl(dataValidator);
+
+        fruitsDao = new FruitsDaoImpl();
+        map = new HashMap<>();
+        map.put("b", new CreateOperation(fruitsDao));
```
Comment (Max-Pochepets) [inline src/test/java/core/basesyntax/service/StoreServiceTest.java:42]: you can have private methods in tests, FYI

File: src/test/java/core/basesyntax/service/file/FileWriterTest.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    private static List<String> expected;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void writeToFIle_correctWrite_Ok() {
+        fileWriter.writeToFile("src/test/resources/writeTest.csv", "Still hate Junit4");
```
Comment (Max-Pochepets) [inline src/test/java/core/basesyntax/service/file/FileWriterTest.java:23]: make path a separate variable, everywhere please

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
good job 👍

File: src/test/java/core/basesyntax/service/StoreServiceTest.java [hunk 6]
```diff
@@ -24,35 +24,35 @@
...
-        dataValidator = new DataValidator();
+        DataValidator dataValidator = new DataValidator();
         parser = new FruitRecordParserServiceImpl(dataValidator);
+        fileReader = new FileReaderImpl();
+        initializeStoreService();
+    }
 
-        fruitsDao = new FruitsDaoImpl();
-        map = new HashMap<>();
+    private static void initializeStoreService() {
```
Comment (Max-Pochepets) [inline src/test/java/core/basesyntax/service/StoreServiceTest.java:41]: keep an eye on methods order, private after public

**Review by Max-Pochepets — APPROVED** [review]
LGTM

**Pull request closed without merging**